scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -22,15 +22,28 @@ Set-Variable DockerServiceName -Value 'com.docker.service' -Option Constant
 
 Set-Variable EdgePackage -Value 'microsoft-azure-iotedge' -Option Constant
 
-Set-Variable EdgeInstallDirectory -Value "$env:ProgramFiles\iotedge" -Option Constant
+# If the user is running a 32-bit PS host on a 64-bit OS, then `$env:ProgramFiles` points to `C:\Program Files (x86)`
```

Comment:
Why do we need to update this file? is there any exception when call runE2ETests.sh?

Reply:
Not sure. I'll revert changes and see.